src/Collision/Graspit/collisionAlgorithms_inl.h

```diff
@@ -187,7 +187,7 @@ void DistanceCallback::leafTest(const Leaf *l1, const Leaf *l2)
 		const std::list<Triangle>*	list2 = l2->getTriangles();
 		for (it2 = list2->begin(); it2!=list2->end() && mMinDistSq >= 0.0; it2++) {
 			mNumTriangleTests++;
-			position p1,p2;
+            position p1=NULL,p2=NULL;
```

Comment:
I don't think NULL is a legal position. Again, perhaps use a valid constructor (e.g. position p1(0,0,0)). Not entirely sure this is necessary though - does the function call below complain about a possibly uninitialized argument?